templates/default.yaml

```diff
@@ -123,6 +123,19 @@ mountInotify: null
 # ADVANCED CONFIGURATION
 # ===================================================================== #
 
+# User to be used inside the VM
```

Comment:
Probably this should be in the `FURTHER ADVANCED CONFIGURATION`  section

Reply:
done